setup.py

```diff
@@ -48,9 +51,10 @@
       url="https://github.com/ncclient/ncclient",
       packages=find_packages('.'),
       install_requires=install_reqs,
+      tests_require=test_reqs,
       license=__licence__,
       platforms=["Posix; OS X; Windows"],
-      keywords=('NETCONF', 'NETCONF Python client', 'Juniper Optimization', 'Cisco NXOS Optimization'),
+      keywords=['NETCONF', 'NETCONF Python client', 'Juniper Optimization', 'Cisco NXOS Optimization'],
```

Comment:
Seems like an odd change - what's the reason for this?

Reply:
run python setup.py test

It will bitch about () vs []